devito/function.py

```diff
@@ -44,26 +43,21 @@ def __repr__(self):
 Backward = TimeAxis('Backward')
 
 
-class Constant(SymbolicFunction, ConstantArgProvider):
+class Constant(AbstractSymbol, ConstantArgProvider):
 
     """
-    Data object for constant values.
+    Symbol representing constant values im symbolic equations.
     """
 
     is_Constant = True
     is_Scalar = True
 
-    def __new__(cls, *args, **kwargs):
-        kwargs.update({'options': {'evaluate': False}})
-        return AbstractSymbol.__new__(cls, *args, **kwargs)
+    is_SymbolicFunction = False
```

Comment:
This derives from ``AbstractSymbol``, so it should be unnecessary to set this to False, right?